### PR TITLE
Feature/capability service

### PR DESCRIPTION
**Pull request by ronin1972**

### CODE REVIEW

File: capabilities/capability-service.yaml [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+        - Manage Assessments
+      operationId: create-assessment
+      requestBody:
+        content:
+          'application/json:':
+            schema:
+              $ref: '#/components/schemas/Assessment'
+      responses:
+        '201':
+          description: JourneyID for the created journey
```
Comment (adrianmilne) [inline capabilities/capability-service.yaml:56]: Looks like a copy/paste from journeys API

File: capabilities/capability-service.yaml [hunk 2]
```diff
@@ -0,0 +1,460 @@
+openapi: "3.0.1"
+info:
+  title: "SCALE Capability Service"
+  version: v0.0.3
```
Comment (adrianmilne) [inline capabilities/capability-service.yaml:4]: Minor point - but maybe should be v0.0.1 for first iteration

File: capabilities/capability-service.yaml [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+      summary: Create a new Assessment
+    get:
+      summary: Return list of assessments.
+      description: Retrieves a list of assessments created by user.
+      tags:
+        - Manage Assessments
+      operationId: get-users-assessments
+      responses:
+        '200':
+          description: Journey State
```
Comment (adrianmilne) [inline capabilities/capability-service.yaml:78]: Journey desc again

File: capabilities/capability-service.yaml [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+        - Manage Requirements
+      operationId: update-requirements
+      requestBody:
+        content:
+          'application/json:':
+            schema:
+              $ref: '#/components/schemas/Requirement'
+      responses:
+        '201':
+          description: JourneyID for the created journey
```
Comment (adrianmilne) [inline capabilities/capability-service.yaml:170]: Journey desc

File: capabilities/capability-service.yaml (hunk 2 above)
Comment (ronin1972) [reply capabilities/capability-service.yaml:4]: Fixed

File: capabilities/capability-service.yaml (hunk 4 above)
Comment (ronin1972) [reply capabilities/capability-service.yaml:170]: Fixed

File: capabilities/capability-service.yaml (hunk 3 above)
Comment (ronin1972) [reply capabilities/capability-service.yaml:78]: Fixed

**Review by adrianmilne — APPROVED** [review]

**Pull request merged**